test/integration/create-next-app/templates-pages.test.ts

```diff
@@ -256,6 +256,11 @@ describe('create-next-app templates', () => {
         ],
         {
           cwd,
+          // @ts-expect-error process.env.NODE_ENV is not always avaliable in create-next-app
```

Comment:
Not sure I understand the `NODE_ENV` part here

Reply:
The TypeScript is warning me about the incompatible type here, as Next.js (in `global.d.ts`) has overwritten `NodeJS.ProcessEnv` to have `NODE_ENV`, which is actually not required here.